crypto/build.rs

```diff
@@ -0,0 +1,17 @@
+extern crate gcc;
+
+use std::env;
+use std::path::Path;
+
+fn main() {
+    gcc::Config::new()
+        .compiler(&Path::new("arm-none-eabi-gcc"))
+        .flag("-mcpu=cortex-m4")
```

Comment:
Initial reaction: Yeah, upwards compatibility is obvious, and if it will work for an m0 and m3 as well, better to be more inclusive.

Caveat: Is it still constant-time on m0 and m3 (esp m0 which has either a 1 or 32-cycle multiply unit, which is arch-independent and chip-specific), or only m4? If it's only constant time on an m4, then I'd be inclined to leave it.

Related Q that I'll bump to top-level thread is how you're verifying constant-time?